### PR TITLE
Add new resource and datasource for Config Aggregator.

### PR DESCRIPTION
**Pull request by liuri1205**

### CODE REVIEW

File: alicloud/data_source_alicloud_config_aggregators.go [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+						},
+						"aggregator_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"description": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"status": {
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_config_aggregators.go:97]: still applies

File: alicloud/resource_alicloud_config_aggregator.go [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+			Create: schema.DefaultTimeout(1 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"aggregator_accounts": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"account_id": {
+							Type:     schema.TypeInt,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_config_aggregator.go:34]: still applies

File: alicloud/data_source_alicloud_config_aggregators.go [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+				ValidateFunc: validation.ValidateRegexp,
+				ForceNew:     true,
+			},
+			"names": {
+				Type:     schema.TypeList,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Computed: true,
+			},
+			"status": {
+				Type:         schema.TypeInt,
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_config_aggregators.go:39]: The status should be converted to string

File: alicloud/resource_alicloud_config_aggregator.go [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"CUSTOM", "RD"}, false),
+				Default:      "CUSTOM",
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"status": {
+				Type:     schema.TypeInt,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_config_aggregator.go:64]: still applies

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/data_source_alicloud_config_aggregators.go [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"aggregators": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"account_id": {
+							Type:     schema.TypeInt,
```
[the remark this answers is not in the document]
Comment (liuri1205) [reply alicloud/data_source_alicloud_config_aggregators.go]: ok

File: alicloud/data_source_alicloud_config_aggregators.go [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+							Computed: true,
+						},
+						"aggregator_accounts": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"account_id": {
+										Type:     schema.TypeInt,
+										Computed: true,
```
[the remark this answers is not in the document]
Comment (liuri1205) [reply alicloud/data_source_alicloud_config_aggregators.go:64]: ok

File: alicloud/data_source_alicloud_config_aggregators.go (hunk 1 above)
Comment (liuri1205) [reply alicloud/data_source_alicloud_config_aggregators.go:97]: ok

File: alicloud/resource_alicloud_config_aggregator.go (hunk 2 above)
Comment (liuri1205) [reply alicloud/resource_alicloud_config_aggregator.go]: ok

File: alicloud/data_source_alicloud_config_aggregators.go (hunk 3 above)
Comment (liuri1205) [reply alicloud/data_source_alicloud_config_aggregators.go:39]: ok

File: alicloud/resource_alicloud_config_aggregator.go (hunk 4 above)
Comment (liuri1205) [reply alicloud/resource_alicloud_config_aggregator.go:64]: ok

**Pull request merged**